conf/muchos.props.example.azure

```diff
@@ -0,0 +1,180 @@
+azure_proxy_host =
+location = westus2
+# Optional Azure fileshare to mount on all nodes.
+# Path and credentials must be updated to enable this.
+#azure_fileshare_mount = /mnt/azure-fileshare
+#azure_fileshare = //fileshare-to-mount.file.core.windows.net/path
+#azure_fileshare_username = fs_username
+#azure_fileshare_password = fs_password
+# Optional integration with Azure Log Analytics
+# Workspace ID and key must be updated to enable this.
+az_omsIntegrationNeeded = False
+#az_logs_id = workspace_id
+#az_logs_key = workspace_key
+
+[existing]
+# Root of data dirs
+mount_root = /var/data
+# Data directories on all nodes
+data_dirs = /var/data1,/var/data2,/var/data3
+# Identifies drives for metrics
+metrics_drive_ids = var-data1,var-data2,var-data3
+
+[performance]
+# Automatically tune Accumulo, Yarn, and Fluo performance setting by selecting or
+# creating a performance profile.  Try not to use more memory than each node has
+# and leave some space for the OS.
+profile=azd8s
+
+# Below are different performance profiles that can be selected.  Each profile
+# has the same properties with different values.
+
+[perf-small]
+# Amount of JVM heap for each tserver
+accumulo_tserv_mem=2G
+# Amount of data cache for each tserver. Only applies when using Accumulo 1.x
+accumulo_dcache_size=768M
+# Amount of index cache for each tserver. Only applies when using Accumulo 1.x
+accumulo_icache_size=256M
+# In memory map size for each tserver. Only applies when using Accumulo 1.x
+accumulo_imap_size=512M
+# Amount of JVM heap for each Fluo worker
+fluo_worker_mem_mb=2048
+# Determines the gap between the Yarn memory limit and the java -Xmx setting.
+# For example if fluo_worker_mem_mb is set to 2048 and twill_reserve_mem_mb is
+# set to 256, then for workers the java -Xmx setting will be set to 2048-256.
+# If yarn is killing worker processes because they are using too much memory,
+# then consider increasing this setting.
+twill_reserve_mem_mb=256
+# Number of threads for each Flup worker
+fluo_worker_threads=20
+# Number of worker to run per node
+fluo_worker_instances_multiplier=1
+# Max amount of memory for YARN per node
+yarn_nm_mem_mb=4096
+
+[perf-medium]
+accumulo_tserv_mem=3G
+# Accumulo configs below only apply when using Accumulo 1.x
+accumulo_dcache_size=1536M
+accumulo_icache_size=512M
+accumulo_imap_size=512M
+fluo_worker_mem_mb=4096
+twill_reserve_mem_mb=512
+fluo_worker_threads=64
+fluo_worker_instances_multiplier=1
+yarn_nm_mem_mb=8192
+
+[perf-large]
+accumulo_tserv_mem=4G
+# Accumulo configs below only apply when using Accumulo 1.x
+accumulo_dcache_size=2G
+accumulo_icache_size=1G
+accumulo_imap_size=512M
+fluo_worker_mem_mb=4096
+twill_reserve_mem_mb=512
+fluo_worker_threads=64
+fluo_worker_instances_multiplier=2
+yarn_nm_mem_mb=16384
+
+[azd16s]
+accumulo_tserv_mem=4G
+accumulo_dcache_size=512M
+accumulo_icache_size=512M
+accumulo_imap_size=44G
```

Comment:
We were optimizing for ingest throughput, and the 44GB was based on the associated VM size which has 64GB of RAM. We can reduce the value to a smaller default - @keith-turner what would you recommend?

Reply:
I don't generally give too much memory because compactions are logarithmic, so there is diminishing returns from increasing the memory size.  However I have not run any recent numbers on throughput with different memory sizes, so I can't give any specific recommendations.   I was just curious about why these numbers were chosen.

Reply:
These settings are now identical to the perf-large configuration.